docs/DeveloperGuide.md

```diff
@@ -632,3 +632,20 @@ testers are expected to do more *exploratory* testing.
     1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+## **Appendix: Planned Enhancements**
+
+Team size: 5
+
+1. **Always show details of client being modified**
+
+To improve clarity for users, the details pane should always show the information of the client that is being modified/had just been modified.
```

Comment:
I think we can improve this to be "show the modified details of the client". In that case, it would better match both examples.